extensions/amp-story-360/0.1/amp-story-360.md

```diff
@@ -0,0 +1,224 @@
+
+<figure class="centered-fig">
+  <amp-anim alt="amp-story-player example" width="300" height="533" layout="fixed" src="https://raw.githubusercontent.com/processprocess/amphtml/360-docs/extensions/amp-story-360/img/amp-story-360.gif">
+    <noscript>
+    <img alt="amp-story-player example" src="https://raw.githubusercontent.com/processprocess/amphtml/360-docs/extensions/amp-story-360/img/amp-story-360.gif" />
+  </noscript>
+  </amp-anim>
+</figure>
+
+The `amp-story-360` component provides a way to embed 360 images and videos in [Web Stories](https://amp.dev/documentation/guides-and-tutorials/start/create_successful_stories/?format=stories).
+
+## Usage
+
+Use the `amp-story-360` component to render 360 images and video. The component can animate between two points or be explorable via the device's gyroscope sensor. For best results, only use one element per [`amp-story-page`](https://amp.dev/documentation/components/amp-story-page/?format=stories).
+
+### Media Requirements
+
+The `amp-story-360` component renders videos and images in [equirectangular projection](https://en.wikipedia.org/wiki/Equirectangular_projection).
+This is the default format generated by many 360 cameras and 3D rendering software.
+
+<amp-img src="https://raw.githubusercontent.com/processprocess/amphtml/360-docs/extensions/amp-story-360/img/SeanDoran-Quela-sol1462-edited_ver2-sm_600-300.jpg" layout="intrinsic" width="400" height="230">
+
+Credit: [NASA / JPL / MSSS / Seán Doran](https://informal.jpl.nasa.gov/museum/360-video).
+
+### Dimensions
+
+Use a 2:1 aspect ratio for the best distribution of resolution.
+
+### Size
+
+2048 x 1024 (2k) is recommended for images and video.
+
+### Configuration
+
+The `amp-story-360` component accepts 1 descendant. This descendant must be an [amp-img](https://amp.dev/documentation/components/amp-img/?format=stories) or [amp-video](https://amp.dev/documentation/components/amp-video/?format=stories).
+
+#### amp-image
+
+[sourcecode:html]
+<amp-story-360 layout="fill">
+<amp-img layout="fill" src="assets/image360.jpg"></amp-img>
+</amp-story-360>
+[/sourcecode]
+
+#### amp-video
+
+[sourcecode:html]
+<amp-story-360 layout="fill">
+<amp-video layout="fill" poster="assets/poster.jpg" autoplay loop>
+
+<source src="assets/video360.mp4" type="video/mp4" />
+</amp-video>
+</amp-story-360>
+[/sourcecode]
+
+Be sure to import the `amp-video` component.
+
+#### Defining a initial point of view (PoV)
+
+[sourcecode:html]
+<amp-story-360 
+   layout="fill"
+   heading-start="95" pitch-start="-10">
+...
+</amp-story-360>
+[/sourcecode]
+
+#### PoV to PoV animation
+
+[sourcecode:html]
+<amp-story-360 
+   layout="fill"
+   heading-start="95" pitch-start="-10"
+   heading-end="-45" pitch-end="-20"
+   duration="3s">
+...
+</amp-story-360>
+[/sourcecode]
+
+The `duration` attribute is required for animation.
+
+#### Gyroscope
+
+Use the device gyroscope sensor to set the image's orientation. The component defaults to the `heading-end` value.
```

Comment:
nit: this sort of implies that `heading-end` is required to apply a default heading, which is not the case: if `heading-end` is not provided, `heading-start` is used. If neither are provided the heading will default to 0.